datasets/ONS-UK-trade-in-services/main.py

```diff
@@ -132,46 +56,24 @@ def date_time (date):
         return 'quarter/' + left(date,4) + '-' + right(date,2)
     #quarter/2019-01
     else:
-        return date
-
-
-# %%
-new_table['Period'] = new_table['Period'].astype(str).replace('\.', '', regex=True)
-new_table['Period'] =  new_table["Period"].apply(date_time)
-trace.Period("Formating to be year/0000 and quarter/2019-01 ")
-
-# %%
-new_table = new_table.replace({'Marker' : {'-' : 'itis-nil', '..' : 'disclosive'}})
-trace.Marker("Replcaing - to mean itis-nil and .. to mean disclosive")
-
-# %%
-new_table['Seasonal Adjustment'] =  'NSA'
-trace.Seasonal_Adjustment("Adding in column Seasonal Adjustment with value NSA")
-
-# %%
-new_table['Flow'] = new_table['Flow'].apply(pathify)
-
-# %%
-tidy = new_table[['Period','Flow','Trade Services','Country','Seasonal Adjustment','Value','Marker' ]]
-
-# %%
-tidy.loc[(tidy['Trade Services'] == 'all'),'Trade Services'] = '0'
-tidy['Trade Services'] = tidy['Trade Services'].astype(str).replace('\.0', '', regex=True)
-tidy.loc[(tidy['Marker'] == 'disclosive'),'Value'] = 0
-tidy['Value'] = tidy['Value'].astype(int)
-tidy['Marker'].fillna('', inplace=True)
-
-# %%
-tidy = tidy.drop_duplicates()
-
-# %%
-trace.render("spec_v1.html")
-
-# %%
-cubes.add_cube(scraper, tidy, "ONS UK trade in services by country and partner country" )
-cubes.output_all()
-
-# %%
-tidy.head()
-
-# %%
+        return date       
+
+
+#Post Processing 
+df['Period'] = df['Period'].astype(str).replace('\.', '', regex=True)
+df['Period'] =  df["Period"].apply(date_time)
+df = df.replace({'Marker' : {'-' : 'itis-nil', '..' : 'disclosive'}})
+df['Seasonal Adjustment'] =  'NSA'
+df['Flow'] = df['Flow'].apply(pathify)
+df = df[['Period','Flow','Trade Services','Country','Seasonal Adjustment','Value','Marker' ]]
+df['Trade Services'] = df['Trade Services'].astype(str).replace('\.0', '', regex=True)
+df.loc[(df['Marker'] == 'disclosive'),'Value'] = 0
+df['Value'] = df['Value'].astype(int)
+df['Marker'].fillna('', inplace=True)
+df = df.drop_duplicates()
```

Comment:
Check if there are any duplicates and then you could remove this line of code(df = df.drop_duplicates()) if there aren't any duplicates